packages/ember-views/lib/mixins/view_support.js

```diff
@@ -620,7 +620,7 @@ export default Mixin.create({
     this._super(...arguments);
 
     if (!this.elementId && this.tagName !== '') {
-      this.elementId = guidFor(this);
```

Comment:
We convert the args to a "creation props" pojo that we pass to `Component.create(...)`. That pojo should contain `elementId`, not `id`.